microshift_troubleshooting/microshift-cleanup-data.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-cleanup-data"]
+= Data cleanup
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-cleanup-data
+
+toc::[]
+
+{microshift-short} provides the `microshift-cleanup-data` script for various troubleshooting tasks, such as cleaning or deleting all data, certificates, and container images.
```

Comment:
"cleaning or deleting"? Should it be both or just one?
I'd say deleting is more suitable.